website/blog/2018-07-01-1.14.0.md

```diff
@@ -103,7 +103,10 @@ content
 
 #### Do not put values on a separate line from the key ([#4852] by [@ikatyang])
 
-TODO
+Putting long values on new rows is the core feature of Prettier,
+but we found that it won't improve readability for string value in object,
+the long values being put onto new rows would still not fit the print width in most of the time.
+In Prettier 1.14, string values in object won't be extended to the next line no matter it fits the print width or not.
```

Comment:
Isn’t this just JSON`{,5}`?

Reply:
Added. I originally thought it's in the JSON section so I didn't mention it.